### PR TITLE
CrossChainStorage

### PR DESCRIPTION
**Pull request by gaoios**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/crustio/Crust-Grants-Program/blob/main/applications/application_template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A Ethereum (USDT/CRU) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md).
- [x] I accept the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) and submit the PR.
- [x] The total funding amount of the project is below USDT $30k.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/crustio/Crust-Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

File: applications/Crosschainstorage.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+CCS AI engine layer provide the core app layer，collect each chain value base on different Chain Feature （like cost，speed, etc) value and user Preference to use AI select best chain when upload files。Also Provide single&simple interface  that reduce complex for end user。Following are RestAPI for main entries for CCS AI engine.
+![image](https://user-images.githubusercontent.com/47582549/124771299-9938d500-df6d-11eb-954d-f393c4caf8fa.png)
+
+* Example data for strategies. Base on each chain & strategy feature to train AI mode for selecting chain against different requirement.
+![image](https://user-images.githubusercontent.com/47582549/124771382-aa81e180-df6d-11eb-9936-5763e0d22d18.png)
+
+* An overview of the technology stack to be used
+CCSUI Mainly use react,ipfs framework/technology stack.  For each 3rd chains, use each chain’s(like Crust,FileCoin,Swarm) API to interact(login, get CID, upload etc)
+CCS AI engine use nodejs,js technology to develop AI selector.
+![image](https://user-images.githubusercontent.com/47582549/124771461-b7063a00-df6d-11eb-8267-fa20fd41202e.png)
```
Comment (yyd106) [inline applications/Crosschainstorage.md:34]: The picture looks good, but the text part shows "CSS", is that a typo?

File: applications/Crosschainstorage.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+* Payment Address: 0x17389d755cb720e5e7e12d0fb926c0d53a2846d6 (USDT)
+* Receivable Token: USDT – ERC20
+
+
+### Overview
+
+* A project name: CCS - Cross Chain Storage Platform
+* A brief description of your project: In the ear of data explosion and related storage request, we want to create a decentralized, highly motivated storage platform. We will provide customer (individual and business) with a more secure, privacy-protected and cost-effective data storage service. 
+* CCS will build multiple connection adapter incentive layers; Crust will be the pilot and main storage channel layer for CCS.
+* Our team sees an opportunity to consolidate multiple decentralized storage providers to build a platform like “去哪儿” in air-tickets booking network, which provide business clients with most optimized data / files storage solutions.
```
Comment (yyd106) [inline applications/Crosschainstorage.md:14]: What is the difference between this project and Cold Stack

Comment (yyd106) [inline applications/Crosschainstorage.md:14]: What is the difference between this project and Cold Stack

**Review by yyd106 — COMMENTED** [review]
What is the difference between this project and Cold Stack

File: applications/Crosschainstorage.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+# Crust Grant Proposal
+
+* Project Name: CCS - Cross Chain Storage Platform
+* Team Name: Zhu, Wenfeng (朱文峰); Gao, Le (高乐); Wang, Ruijun (王瑞俊); Sang, Junjun (桑俊俊); Dong, Yuansheng (董远胜); Liu, Qi (刘琦)
+* Payment Address: 0x17389d755cb720e5e7e12d0fb926c0d53a2846d6 (USDT)
+* Receivable Token: USDT – ERC20
+
```
[the remark this answers is not in the document]
Comment (wruijun) [reply applications/Crosschainstorage.md:7]: 1, For users, there is a better UI experience, which is more in line with current customers' usage habits, shields many blockchain usage thresholds (such as wallets, etc.), and lowers migration costs. Coldstack currently directly connects to the neutralization layer in one step, exposing a lot of complexity to customers and increasing the cost of use.

2, On the customer side: CCS currently focuses on the needs of corporate customers as the first step, then in the future will  develops individual users version. At the same time, engage domestic market first and then globalwide. Coldstack pays more attention to individual users, most of them are foreign customers.

3, Crust integration: coldstack has not yet done crust integration, and is still planning. CCS has taken the lead in completing the MVP verification of crust integration. Crust is the first choice.

File: applications/Crosschainstorage.md (hunk 2 above)
Comment (wruijun) [reply applications/Crosschainstorage.md:14]: 1, For users, there is a better UI experience, which is more in line with current customers' usage habits, shields many blockchain usage thresholds (such as wallets, etc.), and lowers migration costs. Coldstack currently directly connects to the neutralization layer in one step, exposing a lot of complexity to customers and increasing the cost of use.

2, On the customer side: CCS currently focuses on the needs of corporate customers as the first step, then in the future will  develops individual users version. At the same time, engage domestic market first and then globalwide. Coldstack pays more attention to individual users, most of them are foreign customers.

3, Crust integration: coldstack has not yet done crust integration, and is still planning. CCS has taken the lead in completing the MVP verification of crust integration. Crust is the first choice.

File: applications/Crosschainstorage.md (hunk 1 above)
Comment (wruijun) [reply applications/Crosschainstorage.md:34]: thanks for your question
CCS in our context standards for "Cross Chain Storage" which is our project name, not a typo.

File: applications/Crosschainstorage.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (gaoios) [reply applications/Crosschainstorage.md:7]: I will resubmit a new pull.

File: applications/Crosschainstorage.md (hunk 1 above)
Comment (yyd106) [reply applications/Crosschainstorage.md:34]: I saw both CCS and CSS from https://user-images.githubusercontent.com/47582549/124771461-b7063a00-df6d-11eb-8267-fa20fd41202e.png

Comment (wruijun) [reply applications/Crosschainstorage.md:34]: Sorry, it's a wrong typing from our engineer

File: applications/Crosschainstorage.md (hunk 2 above)
Comment (yyd106) [reply applications/Crosschainstorage.md:14]: I have dropped an email to you, could we have a meeting?

**Pull request closed without merging**